HLTrigger/Tools/bin/hltTimingSummary.cpp

```diff
@@ -1880,16 +1880,16 @@ int main(int argc, char ** argv) {
       }
     }
     
-    char value[10] ; 
+    char value[330] ; 
```

Comment:
> The buffer needs to be big, because we are printing `double` here

True, but we it looks like we are printing bin counts and/or execution times here... if the execution time needs 300 characters to print, it's unlikely I'll stick around to see it printed :-)